logic.js

```diff
@@ -0,0 +1,108 @@
+// variables
+const btnAddCard = document.querySelector('button');
+const popUpWindow = document.querySelector('.popup');
+const iconeClose = document.querySelector('.fa-times-circle');
+const addCardForm = document.querySelector('.addCard');
+const container = document.querySelector('.container');
+const lis = document.getElementsByTagName('li');
+let counter = 0;
+let change = false;
+
+// functions
+function addCard(e) {
+    e.preventDefault()
+    let cardName = this.querySelector('#cardName').value;
+    this.reset();
+    popUpWindow.classList.remove('popupDisplay');
+    container.append(createElement(cardName))
+}
+
+function createElement(cardName) {
+    // main container
+    const divContainer = document.createElement('div');
+    divContainer.classList.add('wrapper');
+    // close icon
+    const closeIcon = document.createElement('i');
+    closeIcon.classList.add('far', 'fa-times-circle', 'closeCard');
+    divContainer.append(closeIcon);
+    // title
+    const h2 = document.createElement('h2');
+    h2.classList.add('title');
+    h2.textContent = cardName
+    divContainer.append(h2);
+    // hr
+    const hr = document.createElement('hr');
```

Comment:
Rename to something more descriptive. As a general note, try to avoid naming your variables the same names as html elements - try and make them more descriptive :)